packages/core/src/events/EventManager.ts

```diff
@@ -85,7 +89,7 @@ export class EventManager {
   }
 
   clone() {
-    return new EventManager(this.subscribers);
+    return new EventManager([...this.subscribers.values()]);
```

Comment:
```suggestion
    return new EventManager([...this.subscribers]);
```

(the copy isnt really necessary, you could as well just mark the constructor parameter as readonly and accept any `Iterable`)